src/platforms/zephyr/src/libAtomVM/CMakeLists.txt

```diff
@@ -0,0 +1,174 @@
+#
```

Comment:
Zephyr official answer: https://github.com/zephyrproject-rtos/zephyr/discussions/66683

Short version: we muse use `ExternalProject_Add`